tests/bench_http_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build !compile
```

Comment:
I think we don't need this, what ever has postfix `_test` is excluded from build.